seg_lapa/config_parse/callbacks_conf.py

```diff
@@ -40,15 +41,25 @@ def get_callback(self):
         return EarlyStopping(**args_dict)
 
 
+@dataclass(frozen=True)
+class CheckpointConf:
+    """Dataclass just to initialize and return the Checkpoint Callback"""
+
+    def get_callback(self, logs_dir):
```

Comment:
```suggestion
class CheckpointConf:
    """Dataclass just to initialize and return the Checkpoint Callback"""
    monitor: Optional[str]
    mode: str
    save_last: Optional[bool]
    period: int
    

    def get_callback(self, logs_dir):
```
Please add the other arguments to Modelcheckpoint to the .yaml config file and to this Conf class. These are important parameters to be able to control. 
In our case, we want to monitor `Val/loss` and mode should be `max`.

https://pytorch-lightning.readthedocs.io/en/latest/generated/pytorch_lightning.callbacks.ModelCheckpoint.html#pytorch_lightning.callbacks.ModelCheckpoint